src/floating-focus.scss

```diff
@@ -10,10 +10,12 @@
 	overflow: hidden;
 	z-index: 9999999999; // It should always be on top of everything, no matter what.
 
-	&.moving {
-		transition-property: opacity, left, top, width, height, border-width, border-radius;
-		transition-duration: .2s, .1s, .1s, .1s, .1s, .1s, .1s;
-		transition-timing-function: linear, ease, ease, ease, ease, ease, ease;
+	@media (prefers-reduced-motion: no-preference) {
```

Comment:
Wat als `prefers-reduced-motion` niet beschikbaar is in de browser?

Reply:
Dan hebben we geen transitions. Wat de meest safe bet is, toch? Zeker in een a11y tool zou de meest a11y state de default moeten zijn IMHO.

Reply:
Tenzij de transition het juist makkelijker moet maken om te zien waar de focus heen springt natuurlijk 🤔 Weten we wat daadwerkelijke gebruikers daarvan vinden? Het punt van de gebruiker die dit aankaartte was juist dat de default zonder transitions zou moeten zijn. Maar dat is n=1 natuurlijk.

Reply:
Dat zou dan voor IE zijn.

Is dit niet iets netter?

```
@media (prefers-reduced-motion: reduce) {
  &.moving {
    transition: none;
  }
}
```

Reply:
👍